### PR TITLE
Fixed missing aria label on text search input.

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://issues.redhat.com/browse/RHCLOUD-6077

### Changes
- added default prop 'aria-label' to text

### CODE REVIEW

**Review by karelhala — APPROVED** [review]

**Comment by Hyperkid123** [comment]
@karelhala oh I have not even noticed that. I just run npm install as usual. Let me try again.

**Comment by Hyperkid123** [comment]
@karelhala I have reinstalled dependencies and the lock file change is still there. Possibly it may be just my version of node and npm. Or we forgot it to add it in some previous PR. Can you check locally if you have these after running the install on master?

**Comment by Hyperkid123** [comment]
@karelhala I have restored the lock file to master. We can deal with it later.
